zerver/models.py

```diff
@@ -307,7 +307,9 @@ class Realm(models.Model):
         "Stream", related_name="+", null=True, blank=True, on_delete=CASCADE,
     )
 
-    RETAIN_MESSAGE_FOREVER = -1
+    MESSAGE_RETENTION_SPECIAL_VALUES_MAP = {
+        'forever': -1,
```

Comment:
`-1` is not a valid value for `Realm.message_retention_days`, this should be `None`, or did I miss some changes that turned things around?

Reply:
I think that's correct.